build-and-deploy.py

```diff
@@ -49,7 +69,9 @@
     print("Building...")
 
     ## Start building:
-    subprocess.run(["yarn", "build"], env={**os.environ, "PUBLIC_URL": url_path})
+    subprocess.run(
+        ["yarn", "build"], env={**os.environ, "PUBLIC_URL": url_path}, check=True
```

Comment:
when yarn build failed (returning non-zero exit code), it did not echo it to our primary process.  Based on the docs, adding `check=True` should throw an error when non-zero exit codes. 

Is this the correct approach @vst

Reply:
That is much correct, Sir!